functions.js

```diff
@@ -146,7 +146,7 @@ Output:
 */
 
 export function getLastItem(array) {
-    return true;
+    return array[3];
```

Comment:
There is a better way to solve this problem but glad to see you're understanding array structure!